05_stringer/NimishMudhale/main.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+type IPAddr [4]byte
+
+func (address IPAddr) String() string {
+	return fmt.Sprintf("%v.%v.%v.%v", address[0], address[1], address[2], address[3])
```

Comment:
You shouldn't use `%v` in a format string if you know the inputs are a literals. Use `%d` for integer types instead.